doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,83 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, updates received over a wired connection (such as Ethernet)
+are still commonly referred to as OTA updates. This approach requires server
+infrastructure to host the firmware binary and implement a method of signaling
+when an update is available. Security is a concern with OTA updates; firmware
+binaries should be cryptographically signed and verified before flashing.
+
+The :ref:`dfu` section discusses upgrading Zephyr firmware using MCUboot. The
+same method can be used as part of OTA. The binary is first downloaded
+into an unoccupied code partition, then flashed using the `MCUboot bootloader`_
+process.
+
+Examples of OTA
+***************
+
+Golioth
+=======
+
+`Golioth`_ is an IoT management platform that includes OTA updates. Devices are
+configured to observe your available firmware revisions on the Golioth Cloud.
+When a new version is available, the device downloads and flashes the binary. In
+this implementation, the connection between cloud and device is secured using
+TLS/DTLS, and the signed firmware binary is confirmed by MCUboot before flashing
+occurs.
+
+1. A working sample can be found on the `Golioth Zephyr-SDK repository`_
+2. The `Golioth OTA documentation`_ includes complete information about the
+   versioning process
+
+Eclipse hawkBit |trade|
+=======================
+
+`Eclipse hawkBit`_ |trade| is an update server framework that uses polling on a
+REST api to detect firmware updates. When a new update is detected, the binary
+is downloaded and MCUboot verifies the signature before flashing the firmware.
+
+There is a :ref:`hawkbit-api-sample` included in the
+Zephyr :ref:`mgmt-samples` section.
+
+UpdateHub
+=========
+
+`UpdateHub`_ is a platform for remotely updating embedded devices. Updates can
+be manually triggered or monitored via polling. When a new update is detected,
+the binary is downloaded and MCUboot verifies the signature before flashing the
+firmware.
```

Comment:
Similarly, does updatehub require mcuboot, or is that an implementation decision in upstream zephyr's updatehub integration?

Reply:
I don't know the answer to this; the sample uses MCUboot. I think your wording from the previous comment also makes sense here.